curriculum/challenges/_meta/learn-data-structures-by-building-the-merge-sort-algorithm/meta.json

```
@@ -0,0 +1,149 @@
+{
+  "name": "Learn Data Structures by Building the Merge Sort Algorithm",
+  "isUpcomingChange": true,
+  "usesMultifileEditor": true,
+  "hasEditableBoundaries": true,
+  "dashedName": "learn-data-structures-by-building-the-merge-sort-algorithm",
+  "order": 42,
```

```suggestion
  "order": 2,
```
Change this to please the tests

---

Done